### PR TITLE
uploaded Assignment 5 and 6

### PR DESCRIPTION
**Pull request by Prashant698**

### CODE REVIEW

File: prashant/HTML/Assignment5/AboutMe.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            <ul>
+                <li>Playing Cricket</li>
+                <li>Love to Code</li>
+                <li>Internet Suffering</li>
+                <li>Reading Books</li>
+            </ul><br>
+            <li>Photo Gallery</li>
+        </ul>
+    </body>
+</html>
```
Comment (sourabh-kanwade) [inline prashant/HTML/Assignment5/AboutMe.html]: add extra line after html

Comment (sourabh-kanwade) [reply prashant/HTML/Assignment5/AboutMe.html]: for all files

**Review by sourabh-kanwade — CHANGES_REQUESTED** [review]

**Review by dayanandviveki — APPROVED** [review]
No change Requred

File: prashant/HTML/Assignment5/AboutMe.html (hunk 1 above)
Comment (Prashant698) [reply prashant/HTML/Assignment5/AboutMe.html]: > add extra line after html

where

File: prashant/HTML/Assignment6/form.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            <label id="skill" for="skill">Skill Set : </label>
+            <input type="checkbox" name="skill" value="HTML">
+            <label for="html">HTML</label>
+            <input type="checkbox" name="skill" value="Java Script">
+            <label for="Java Script">Java Script</label>
+            <input type="checkbox" name="skill" value="Java">
+            <label for="Java">Java</label>
+            <input type="checkbox" name="skill" value="Oracle">
+            <label for="Oracle">Oracle</label>
+
```
Comment (Pawan1005) [inline prashant/HTML/Assignment6/form.html]: remove extra line

File: prashant/HTML/Assignment6/for.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+}
+.input2{
+    margin-top: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+}
+#skill{
+    font-weight: bold;
+}
+
```
Comment (Pawan1005) [inline prashant/HTML/Assignment6/for.css]: remove extra line

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by sourabh-kanwade — APPROVED** [review]

File: prashant/HTML/Assignment6/for.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+}
+#acdmic{
+    margin-bottom: 20px;
+}
+.btn{
+    margin-top: 15px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```
Comment (Pawan1005) [inline prashant/HTML/Assignment6/for.css:44]: add one space before {

File: prashant/HTML/Assignment6/form.html [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+            <fieldset>
+                <legend>Resume</legend>
+                <label for="resume" class="skill">Upload Resume:</label>
+                <input id="resume" type="file" />
+                <br />
+                <label for="comment" class="skill">Comments</label>
+                <textarea name="comment" id="comment" cols="30" rows="5" class="skill"></textarea>
+            </fieldset>
+            <div id="submit-btn" class="btn">
+                <button>Submit</button>
```
Comment (Pawan1005) [inline prashant/HTML/Assignment6/form.html:86]: closing div missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: prashant/HTML/Assignment6/for.css (hunk 4 above)
Comment (Prashant698) [reply prashant/HTML/Assignment6/for.css:44]: done sir

File: prashant/HTML/Assignment6/form.html (hunk 5 above)
Comment (Prashant698) [reply prashant/HTML/Assignment6/form.html:86]: done sir

File: prashant/HTML/Assignment5/aboutme.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    border: darkslateblue 3px;
+}
+.div-self-description{
+    border: solid black 3px;
+    padding: 30px;
+    padding-bottom: 100px;
+    margin-right: 40px;
+    margin-top: 40px;
+    margin-bottom: 0.1px;
+}
```
Comment (Pawan1005) [inline prashant/HTML/Assignment5/aboutme.css:30]: add space before {

File: prashant/HTML/Assignment6/for.css [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
```
Comment (Pawan1005) [inline prashant/HTML/Assignment6/for.css:44]: add space before {

**Review by Pawan1005 — APPROVED** [review]

File: prashant/HTML/Assignment5/AboutMe.html [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+        <div class="div-self-description">    
+            <p>My native place is Jabalpur MadhyaPradesh.</p>
+            <p>I have completed BE from Global Engineering Collage and Schooling from ST Mary's H.S. School.</p>
+            <p> we are 6 members in my family. My father is a Business man, my mother is a house wife, my 2 elder sister is maried and my younger sister was doing Bcom</p>
+        </div> 
+        <p class="p-more-about">MoreAbout Me</p>
+        <ul class="div-self-description">
+            <li>My Resume</li><br>
+            <li>Hobbies</li><br>
+            <ul>
```
Comment (Pawan1005) [inline prashant/HTML/Assignment5/AboutMe.html]: indentation missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: prashant/HTML/Assignment5/aboutme.css (hunk 6 above)
Comment (Prashant698) [reply prashant/HTML/Assignment5/aboutme.css:30]: done sir

File: prashant/HTML/Assignment5/AboutMe.html (hunk 8 above)
Comment (Prashant698) [reply prashant/HTML/Assignment5/AboutMe.html]: done sir

File: prashant/HTML/Assignment6/for.css (hunk 7 above)
Comment (Prashant698) [reply prashant/HTML/Assignment6/for.css:44]: done sir

File: prashant/HTML/Assignment6/for.css (hunk 4 above)
Comment (Pawan1005) [reply prashant/HTML/Assignment6/for.css:44]: check first what i have suggested there

File: prashant/HTML/Assignment5/AboutMe.html (hunk 8 above)
Comment (Pawan1005) [reply prashant/HTML/Assignment5/AboutMe.html]: not done you need to give space before ul tag li is parent tag there and ul is child tag

File: prashant/HTML/Assignment5/aboutme.css [hunk 9]
```diff
@@ -0,0 +1,31 @@
+body{
```
Comment (Pawan1005) [inline prashant/HTML/Assignment5/aboutme.css]: after body tag you need to give one space

Comment (Pawan1005) [reply prashant/HTML/Assignment5/aboutme.css]: for every css tag

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: prashant/HTML/Assignment5/AboutMe.html (hunk 8 above)
Comment (Prashant698) [reply prashant/HTML/Assignment5/AboutMe.html]: ok sir now i have done it please cheack

File: prashant/HTML/Assignment5/aboutme.css (hunk 9 above)
Comment (Prashant698) [reply prashant/HTML/Assignment5/aboutme.css]: ok sir now i have done it please cheack

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**